packages/aws-cdk/lib/convert-to-cli-args.ts

```diff
@@ -254,3 +254,197 @@ export function convertToCliArgs(args: any): CliArguments {
 
   return cliArguments;
 }
+
+// @ts-ignore TS6133
+export function convertConfigToCliArgs(args: any): CliArguments {
```

Comment:
This would probably help in terms of naming. 
```suggestion
export function convertConfigToCliArgs(config: any): CliArguments {
```